src/plugins/GUITestBase/src/GUITestBasePlugin.cpp

```diff
@@ -105,8 +104,25 @@
 #include "tests/regression_scenarios/GTTestsRegressionScenarios_5001_6000.h"
 #include "tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.h"
 
-static QStringList labels(const QString &label) {
-    return QStringList() << label;
+namespace U2 {
+
+/** Converts list of label args into QStringList. */
+static QStringList labels(const QString &label1, const QString &label2 = "", const QString &label3 = "", const QString &label4 = "", const QString &label5 = "") {
```

Comment:
Hardcoded parameters will cause additional unnecessary changes if additional labels appear. I offer to receive a QStringList as parameter (or QList<QString>, if compiler will comply) and initialize the list where the method calls. For example: labels({Metagenomics, Linux})).

Reply:
Good suggestion. I was not aware of this syntax. Fixed.